src/system/RoutesModule/Resources/views/Config/config.html.twig

```diff
@@ -7,7 +7,7 @@
         {% form_theme form with [
             '@ZikulaRoutesModule/Form/bootstrap_4.html.twig',
             '@ZikulaFormExtension/Form/form_div_layout.html.twig'
-        ] %}
+        ] only %}
```

Comment:
why isn't Routes using same as all the others?

Reply:
Because MOST modules have their own template (inheriting from the default one) because they often declare additional form field types.